src/components/messages.js

```diff
@@ -637,6 +637,16 @@ const messages: { [string]: MessageDescriptor } = defineMessages({
         description: 'Error message shown when pending app folder size exceeds the limit',
         defaultMessage: 'Pending app folder size limit exceeded',
     },
+    uploadsProvidedFolderNameInvalidMessage: {
+        id: 'be.uploadsProvidedFolderNameInvalidMessage',
+        description: 'Error message shown when pending folder upload contains invalid characters',
+        defaultMessage: 'Provided folder name, {name}, could not be used to create a folder.',
```

Comment:
Wondering if a better/simpler message would be

`Could not create a folder with the name - {name}`

Def also check if the uploader is small width, text doesn't weirdly overflow.

Reply:
This was to preserve consistency, will explore other options.